test/broadcast.jl

```diff
@@ -151,22 +151,22 @@ end
     @testset "eltype after broadcast" begin
         # test cases issue #198
         let a = SVector{4, Number}(2, 2.0, 4//2, 2+0im)
-            @test_broken eltype(a + 2) == Number
-            @test_broken eltype(a - 2) == Number
-            @test_broken eltype(a * 2) == Number
-            @test_broken eltype(a / 2) == Number
+            @test eltype(a + 2) == Number
```

Comment:
Shouldn't these methods be deprecated? They are for general arrays.
```
julia> [1, 2, 3] + 1
ERROR: MethodError: no method matching +(::Array{Int64,1}, ::Int64)
Closest candidates are:
  +(::Any, ::Any, ::Any, ::Any...) at operators.jl:502
  +(::Complex{Bool}, ::Real) at complex.jl:292
  +(::Missing, ::Number) at missing.jl:93
  ...
Stacktrace:
 [1] top-level scope at none:0

julia> @SVector([1, 2, 3]) + 1
3-element SArray{Tuple{3},Int64,1,3}:
 2
 3
 4
```

Reply:
True, but orthogonal to this issue, except that I could use the explicit broadcast in these tests as I'm touching them anyways. (And it's in the broadcast tests.)

Reply:
Yea, just wanted to point it out since I noticed it :slightly_smiling_face:

Reply:
Updated the tests locally; will push once I have feedback what to do about the empty case.